aspnetcore/tutorials/first-mvc-app/start-mvc/sample/MvcMovie90/Models/Movie.cs

```diff
@@ -117,16 +117,15 @@ public class Movie
 {
     public int Id { get; set; }
     [StringLength(60, MinimumLength = 3)]
-    public string Title { get; set; }
+    public string? Title { get; set; }
     [Display(Name = "Release Date"), DataType(DataType.Date)]
     public DateTime ReleaseDate { get; set; }
     [RegularExpression(@"^[A-Z]+[a-zA-Z\s]*$"), Required, StringLength(30)]
-    public string Genre { get; set; }
-    [Range(1, 100), DataType(DataType.Currency)]
-    [Column(TypeName = "decimal(18, 2)")]
+    public string? Genre { get; set; }
```

Comment:
```suggestion
    public string Genre { get; set; }
```

Reply:
This particular property `Genre` was demonstrating the `[Required]` attribute, while the rest do not.  For `Genre` I think the intent was that it is not nullable.  Whereas the rest should have been explicitly indicated as nullable with the nullable type indicator "?".  Thanks for the correction in the rest!